components/automate-gateway/api/iam/v2/introspect.pb.policy.go

```diff
@@ -9,13 +9,13 @@ import (
 )
 
 func init() {
-	policy.MapMethodTo("/chef.automate.api.iam.v2.Authorization/IntrospectAll", "iam:introspect", "iam:introspect:getAll", "GET", "/iam/v2/introspect", func(unexpandedResource string, input interface{}) string {
+	policy.MapMethodTo("/chef.automate.api.iam.v2.Authorization/IntrospectAll", "iam:introspect", "iam:introspect:getAll", "GET", "/apis/iam/v2/introspect", func(unexpandedResource string, input interface{}) string {
```

Comment:
💡 This is another, uhm, side-effect I hadn't kept in mind. The API paths are used for introspection. I'm not sure if this will _just continue to work_ or if there's extra hoops to jump through. @msorens What do you think?

Reply:
Good catch! I am thinking this will probably fail but will have to take a closer look.